README.md

```diff
@@ -14,6 +14,8 @@ This module requires the following modules/libraries:
 
 **NOTE**: The collection solution pack either requires that the [resource index](https://wiki.duraspace.org/display/FEDORA37/Resource+Index) is enabled in Fedora or you have the [Islandora Solr](https://github.com/islandora/islandora_solr_search) module installed to perform its queries.
 
+**NOTE**: After enabling the display of object metadata, it is possible that some metadata (e.g. Description) will not display if the [Islandora Context](https://github.com/SFULibrary/islandora_context) module is not enabled.
```

Comment:
I'm not sure what enabling Islandora Context does to fix this issue, but I'm reluctant to mention Context in the readme without more.... context. Without knowing any specifics of the issue, this sounds like a problem with Islandora Context, not the Collections SP.

@pivot5 can you provide a description of what the issue is?

Reply:
Apologies, I should have provided more information. On a site with Collections Solution Pack settings as follows:

![image](https://user-images.githubusercontent.com/31073143/93618287-0e1e2c80-f9a5-11ea-84f6-5bf52cf05190.png)

The description is of course not set up to be hidden from display, however it doesn't show up on the actual site while everything else (metadata, content, collections) shows up normally. This applied to all collections on the site with descriptions. This occurred while the Islandora Context module was disabled - please correct me if I'm wrong but I don't think a module that's turned off can affect the solution pack like this unless the solution pack has some form of dependency on the module? Inexplicably when the Islandora Context module was turned on, the description was visible as expected.